io_scene_godot/converters/mesh.py

```diff
@@ -155,63 +180,18 @@ def init_mesh_bones_data(self, skeleton_node):
                 self.vgroup_to_bone_mapping[group.index] = bone_info.id
 
     def export_mesh(self, escn_file, export_settings):
-        """Saves a mesh into the escn file """
-        # Check if it exists so we don't bother to export it twice
-        mesh = self.object.data
-        mesh_id = escn_file.get_internal_resource(mesh)
-
+        """Saves a mesh into the escn file"""
+        mesh_converter = MeshConverter(self.object, export_settings)
+        key = (mesh_converter.resource_name, "Mesh")
```

Comment:
do you think it's a good idea if we built a key from mesh resource and all the modifiers applied on it? (e.g. add a new class `MeshRscKey`. It's `__hash__()` method returns a hashable data which has same uniqueness as the exported mesh resource. This hashable data could be a serialization of `self.object.data` and all the modifiers name (seems it is compact).

Reply:
`resource_name` which may built from a random combination of letters still has a possibility to have a collision

Reply:
Yes that makes a lot more sense.
I did consider making a hash key made up of (mesh, modifiers, modifier properties) but dismissed it as being too complicated, but it isn't really!